java-tests/validating/src/testFixtures/proto/spine/test/tools/validate/required_field.proto

```diff
@@ -49,7 +49,7 @@ message Due {
 }
 
 message Combination {
-    option (required_field) = "a1 & a2 | b1 & b2";
+    option (require).fields = "a1 & a2 | b1 & b2";
```

Comment:
Can we please have the simplest test case as well? I.e. 
```
option (require).fields = "a1"
```

Or maybe it's already implemented, I just cannot see it right now.